routes/spec.js

```diff
@@ -998,13 +998,13 @@ Currently the API has a rate limit of **60 requests/minute** and **50,000 reques
           },
         },
         route: () => '/skyblock/items',
-        func: (req, res) => {
-          redis.get('skyblock_items', (err, items) => {
-            if (err) {
-              logger.error(err);
-              return res.status(500);
+        func: (_, response) => {
+          redis.get('skyblock_items', (error, items) => {
+            if (error) {
+              logger.error(error);
+              return response.status(500);
```

Comment:
Let's use the callback here as well for consistency.

Reply:
Fixed!